projects/packages/my-jetpack/src/products/class-social.php

```diff
@@ -127,6 +127,27 @@ public static function get_wpcom_product_slug() {
 		return 'jetpack_social';
```

Comment:
I think we need to change this to an active product slug - maybe `jetpack_social_basic_yearly`? This is the slug the interstitial is going to the cart with

Reply:
Nice catch. Didn't get that one in my local env.